run_scapy

```diff
@@ -1,8 +1,5 @@
 #! /bin/sh
 DIR=$(dirname $0)
-if python --version 2>&1 | grep -q '^Python 2'; then
-    PYTHON=python
-else
-    PYTHON=python2
-fi
+PYTHONDONTWRITEBYTECODE=True
```

Comment:
Shouldn't we keep the comment to remember why we do this?

Reply:
The comment is no longer true, since I've set this also for Python 2. Also, the variable name seems easy to understand; as it says, "don't write bytecode".

Reply:
ok, fair enough